packages/grafana-ui/src/components/Combobox/Combobox.tsx

```diff
@@ -335,7 +388,8 @@ export const Combobox = <T extends string | number>(props: ComboboxProps<T>) =>
     },
   });
 
-  const { inputRef, floatingRef, floatStyles, scrollRef } = useComboboxFloat(items, isOpen);
+  const validItems = items.filter((item): item is ComboboxOption<T> => item !== undefined && typeof item !== 'string');
```

Comment:
Why do we have to do this filtering? Can we take groups into account in `useComboboxFloat` instead?

Reply:
This was added to fix a type error. The groups are within the items so they are taking into account in `useComboboxFloat`.